### PR TITLE
[APP-2881] Add whats new to app view

### PR DESCRIPTION
**Pull request by Culurio**
**What does this PR do?**

This PR aims to add the what's new box to the App View

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] See by commit

**How should this be manually tested?**

  Flow on how to test this or QA Tickets related to this use-case: [APP-2881](https://aptoide.atlassian.net/browse/APP-2881)

**What are the relevant tickets?**

  Tickets related to this pull-request: [APP-2881](https://aptoide.atlassian.net/browse/APP-2881)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 

**Code Review Checklist**

- [ ] Documentation on public interfaces
- [ ] Database changed? If yes - Migration?
- [ ] Remove comments & unused code & forgotten testing Logs
- [ ] Codestyle
- [ ] New Kotlin code has unit tests
- [ ] New flows in presenters unit tests
- [ ] Mappers/Validators with any kind of logic unit tests
- [ ] Functional tests pass

[APP-2881]: https://aptoide.atlassian.net/browse/APP-2881?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[APP-2881]: https://aptoide.atlassian.net/browse/APP-2881?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app-games/src/main/java/com/aptoide/android/aptoidegames/appview/AppViewScreen.kt [hunk 1]
```diff
@@ -394,17 +397,113 @@ fun DetailsView(app: App) {
...
+          style = AGTypography.InputsL,
+        )
+        Text(
+          text = TextFormatter.formatBytes(app.appSize),
+          color = Palette.GreyLight,
+          style = AGTypography.InputsM
+        )
+        Text(
+          modifier = Modifier
+            .fillMaxWidth(),
```
Comment (G0retZ) [inline app-games/src/main/java/com/aptoide/android/aptoidegames/appview/AppViewScreen.kt:475]: Can be on a single line

File: app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/icons/BookmarkStar.kt [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+fun TestBookMarkStarIcon() {
+  Image(
+    imageVector = getBookmarkStar(Color.Black),
+    contentDescription = null,
+    modifier = Modifier.size(240.dp)
+  )
+}
+
+fun getBookmarkStar(color: Color): ImageVector = ImageVector.Builder(
+  name = "BookmarkStar", defaultWidth = 16.0.dp, defaultHeight =
```
Comment (G0retZ) [inline app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/icons/BookmarkStar.kt:29]: Put arguments on separate lines

File: app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/icons/BookmarkStar.kt [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  16.0.dp, viewportWidth = 16.0f, viewportHeight = 16.0f
+).apply {
+  group {
+    path(
+      fill = SolidColor(color), stroke = null, strokeLineWidth = 0.0f,
```
Comment (G0retZ) [inline app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/icons/BookmarkStar.kt:34]: Put arguments on separate lines

File: feature_apps/src/main/java/cm/aptoide/pt/feature_apps/data/AptoideAppsRepository.kt [hunk 4]
```diff
@@ -290,6 +290,10 @@ fun AppJSON.toDomainModel(
   versionCode = this.file.vercode,
   screenshots = this.media?.screenshots?.map { it.url },
   description = this.media?.description,
+  news = if (this.media?.news.isNullOrEmpty()) null
```
Comment (G0retZ) [inline feature_apps/src/main/java/cm/aptoide/pt/feature_apps/data/AptoideAppsRepository.kt:293]: Add curly braces or make this if-else expression in a single line

**Review by G0retZ — CHANGES_REQUESTED** [review]

File: app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/icons/BookmarkStar.kt [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+    imageVector = getBookmarkStar(Color.Black),
+    contentDescription = null,
+    modifier = Modifier.size(240.dp)
+  )
+}
+
+fun getBookmarkStar(color: Color): ImageVector = ImageVector.Builder(
+  name = "BookmarkStar",
+  defaultWidth = 16.0.dp,
+  defaultHeight =
```
Comment (G0retZ) [inline app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/icons/BookmarkStar.kt:31]: I guess this line break is a typo? 🤔

**Review by G0retZ — CHANGES_REQUESTED** [review]

**Review by G0retZ — APPROVED** [review]

File: app-games/src/main/java/com/aptoide/android/aptoidegames/appview/AppViewScreen.kt [hunk 6]
```diff
@@ -397,17 +400,112 @@ fun DetailsView(app: App) {
...
+    modifier = Modifier
+      .fillMaxWidth()
+      .padding(start = 16.dp, end = 16.dp, top = 24.dp, bottom = 16.dp)
+      .background(Palette.GreyDark)
+  ) {
+    Column(
+      horizontalAlignment = Alignment.CenterHorizontally
+    ) {
+      LatestBox()
+      Row(
```
Comment (pedroagostinhoaptoide) [inline app-games/src/main/java/com/aptoide/android/aptoidegames/appview/AppViewScreen.kt:460]: To fully match the design, the last text could be aligned to be centered vertically and the first two texts could be aligned by their baselines (with the `alignByBaseline` modifier)

**Review by pedroagostinhoaptoide — CHANGES_REQUESTED** [review]

**Review by pedroagostinhoaptoide — APPROVED** [review]

**Pull request merged**